docs-src/releases.md

```diff
@@ -0,0 +1,65 @@
+<!--
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Releases
+
+Although Service APIs are an official Kubernetes project, and represent official
+APIs, these APIs will not be installed by default on Kubernetes clusters. This
+project will use custom resource definitions (CRDs) to represent the new API
+types that Service APIs include.
+
+Similar to other Kubernetes APIs, these will go through a formal Kubernetes
+Enhancement Proposal (KEP) review. Unlike other Kubernetes APIs, Service API
+releases will be independent from Kubernetes releases.
+
+Service API releases will include four components:
+- Custom resource definitions to define the API
+- Go client libraries
+- Validation webhooks to implement cross field validations
+- Conversion webhooks to convert resources between API versions
+
+## Versioning
+
+Versioning will be completely separate from the Kubernetes release process, but
+similar methodology will be used. Service API versions will use the [same
+version level requirements as other Kubernetes
+features](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api_changes.md#alpha-beta-and-stable-versions).
+
+A faster release cadence will be used for Alpha versions, with new alpha
+releases monthly. Users and controller authors will be expected to use the
+latest version of the API. There will be little to no provisions for backwards
+compatibility for alpha versions. Generally we expect the alpha API to be for
+users and controller developers to test out the API but not in any production
+environment.
+
+Beta and stable releases will operate on a slower, more standard, release
+schedule. They will also provide all of the stability guarantees that other beta
+and stable Kubernetes features provide.
+
+## Installation
+
+This project will be responsible for providing straightforward and reliable ways
+to install releases of Service APIs. Notably, that will need to include a
+strategy for the installation of the validation and conversion webhooks.
+
+## Other Official Custom Resources
+
+This is a relatively new concept, and there is only one previous example of
+official custom resources being used being done:
+[VolumeSnapshots](https://kubernetes.io/blog/2018/10/09/introducing-volume-snapshot-alpha-for-kubernetes/).
+Although VolumeSnapshot CRDs can be installed directly by CSI drivers that
+support them, we need to support multiple controllers per cluster, so the CRDs
```

Comment:
s/we need to support/Service APIs must support/

Reply:
Updated, thanks!